runtime/plugin/nvim.lua

```diff
@@ -18,3 +18,11 @@ vim.api.nvim_create_user_command('InspectTree', function(cmd)
     vim.treesitter.inspect_tree()
   end
 end, { desc = 'Inspect treesitter language tree for buffer', count = true })
+
+if vim.g.use_lua_gx ~= nil then
+  vim.keymap.set({ 'n', 'x' }, 'gx', function()
+    local uri = vim.fn.expand(vim.g.netrw_gx ~= nil and vim.g.netrw_gx or '<cfile>')
```

Comment:
The purpose is to avoid netrw so supporting `vim.g.netrw_gx` is out of scope. For now we can just use `<cfile>`, eventually we might want to introduce `<curi>`.